src/testRunner/unittests/typeParameterIsPossiblyReferenced.ts

```diff
@@ -0,0 +1,33 @@
+describe("unittests :: internalApi :: typeParameterIsPossiblyReferenced", () => {
+        it("with type parameter aliasing", () => {
+            const content = `
+            declare function foo<T>(b: T, f: <T>(a: typeof b) => typeof a): typeof f;

```

Comment:
I couldn't come up with a test program such that type checking it observed the consequences of an incorrect `containsReference`/`typeParameterIsPossiblyReferenced` result in the presence of type parameter aliasing, so I wrote a unit test for it.